core/shared/src/main/scala/monocle/std/NonEmptyVector.scala

```diff
@@ -0,0 +1,27 @@
+package monocle.std
+
+import monocle.{Iso, PIso}
+import cats.data.{NonEmptyVector, OneAnd}
+
+import scala.{Vector => IVector}
+
+object nev extends NonEmptyVectorOptics
+
+trait NonEmptyVectorOptics {
+
+  final def pNevToOneAnd[A, B]: PIso[NonEmptyVector[A], NonEmptyVector[B], OneAnd[Vector,A], OneAnd[Vector,B]] =
+    PIso((nev: NonEmptyVector[A])    => OneAnd[Vector,A](nev.head, nev.tail))(
+      (oneAnd: OneAnd[Vector, B]) => NonEmptyVector(oneAnd.head, oneAnd.tail))
+
+  final def nevToOneAnd[A]: Iso[NonEmptyVector[A], OneAnd[Vector,A]] =
+    pNevToOneAnd[A, A]
+
+  final def pOptNevToVector[A, B]: PIso[Option[NonEmptyVector[A]], Option[NonEmptyVector[B]], Vector[A], Vector[B]] =
```

Comment:
it might be useful to add a `Prism` / `PPrism` between `Vector` and `NonEmptyVector `